ceed-fortran.c

```diff
@@ -79,25 +79,22 @@ void fCeedVectorSetArray(int *vec, int *memtype, int *copymode,
 }
 
 #define fCeedVectorGetArray FORTRAN_NAME(ceedvectorgetarray,CEEDVECTORGETARRAY)
-//TODO Need Fixing, double pointer
-void fCeedVectorGetArray(int *vec, int *memtype, CeedScalar *array, int *err) {
+void fCeedVectorGetArray(int *vec, int *memtype, CeedScalar *array, long *offset,
```

Comment:
@jedbrown : Is the use of `long` here for offset okay? Looks like there is no `ssize_t` equivalent in
Fortran?

Reply:
`long` is only 32 bits on LLP64 (e.g., MS Windows). Eventually we'll need to run configure tests to determine sizes, but we would be fine using `int64_t` from `stdint.h` and matching it with 8-byte integer in Fortran.